README.md

```diff
@@ -97,45 +95,91 @@ _Only when accessing a property of this module, would the execution operations b
 This way, `DeferredModuleNamespace` acts like a proxy to the evaluation of the module, effectively
 with getter functions that trigger synchronous idempotent evaluation before returning the defined bindings.
 
-The API can take a couple of forms, here are some suggestions:
+The API will use the below syntax, following the phases model established by the
+[source phase imports](https://github.com/tc39/proposal-source-phase-imports) proposal:
 
 ```js
-// using import attributes (`lazyInit` to illustrated, but it could also be `deferEval`, say)
-import lazyModuleNamespace from "y" with { lazyInit: true }
-
-// with an explicit "*" to indiciate the namespace form
-import * as lazyModuleNamespace from "y" with { lazyInit: true }
-
 // or with a custom keyword:
-lazy import * as yNs from "y";
+import defer * as yNamespace from "y";
 ```
 
-There are other proposals, but the discussion of the naming may derail the discussion of the
-semantics, which should be agreed on first and would likely guide the design of the API.
-
 ## Semantics
 
-The semantics would be similar to import reflection, in that the deferred module namespace can be
-thought of as a leaf node that does not add further branches to the execution graph.
-
 The imports would still participate in deep graph loading so that they are fully populated into
-the module cache prior to execution.
+the module cache prior to execution, however it the imported module will not be evaluated yet.
 
-When a property of a `DeferredModuleNamespace` is accessed, if the execution has not already
-been performed, a new top-level execution would be initiated for that module.
+When a property of the resulting module namespace object is accessed, if the execution has
+not already been performed, a new top-level execution would be initiated for that module.
```

Comment:
Should we mention that reexports are a possible further feature, but initially disallowed?